bin/configs/kotlin-jvm-ktor-kotlinx_serialization.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: kotlin
+outputDir: samples/client/petstore/kotlin-jvm-ktor-kotlinx_serialization
+library: jvm-ktor
+inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore.yaml
```

Comment:
please use 3.0 spec instead

modules/openapi-generator/src/test/resources/3_0/petstore.yaml

Reply:
> please use 3.0 spec instead

I was using 2.0 because all the other kotlin/ktor were using it, but sure, I can update this one.

> Just as a heads-up, I found another issue with ktor + kotlinx-serialization, in regards to not setting the content type, I'll submit another issue/PR as soon as this one gets sorted.

Also, since we are doing changes that are not related strictly to `jsonBlock` anymore, should I add the other fixes I have to this same PR, or do you still prefer that I raise another issue/ticket?

Reply:
>I was using 2.0 because all the other kotlin/ktor were using it, but sure, I can update this one.

yes, i know what you're talking about. we know yet have time to update those to use 3.0 spec.

for new one (test), we definitely want to start with 3.0 spec

Reply:
> Also, since we are doing changes that are not related strictly to jsonBlock anymore, should I add the other fixes I have to this same PR, or do you still prefer that I raise another issue/ticket?

We  prefer small PRs for easier review.

Reply:
Ok, I'll raise another PR right after.

Reply:
Also worth sharing with you that we are gradually moving away from petstore and will switch to echo server instead for client testings. https://github.com/OpenAPITools/openapi-generator/wiki/Integration-Tests#echo-server has more info.

(no need to do anything in this PR. more of an FYI)